pkgs/development/python-modules/pyeasee/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pytest-asyncio,
+  aioresponses,
+  aiohttp,
+  pysignalr,
+}:
+
+buildPythonPackage rec {
+  pname = "pyeasee";
+  version = "0.8.11";
+
+  src = fetchFromGitHub {
+    owner = "nordicopen";
+    repo = "pyeasee";
+    rev = "v${version}";
+    hash = "sha256-bJWSj8dYwQHcgRNAHmCJp68dYr1+Ie2I+TbVJanC5IQ=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    aioresponses
+    pytest-asyncio
+  ];
+
+  dependencies = [
+    aiohttp
+    pysignalr
+  ];
+
+  meta = {
+    description = "Easee EV charger API python library";
+    homepage = "https://github.com/nordicopen/pyeasee";
+    changelog = "https://github.com/nordicopen/pyeasee/releases/tag/v${version}";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ dandellion ];
```

Comment:
Define this without `with` please.

Reply:
This is so narrowly scoped, I think this is a bad recommendation.